src/stores/CatalogOnlieQueryConfigStore.ts

```diff
@@ -317,9 +317,11 @@ export class CatalogOnlineQueryConfigStore {
             const wcsCopy = AST.copy(frame.wcsInfo);
             let astString = new ASTSettingsString();
             const sys = system ? system : overlay.global.explicitSystem ? overlay.global.explicitSystem : SystemType.ICRS;
+            const xAxis = frame.frameInfo.fileInfoExtended.axesNumbers.dirX;
```

Comment:
These error messages are from the AST  function `astSet` in the AST wrapper function `AST.getFormattedCoordinates`. It seems the AST can not accept direction axis numbers that are not 1 or 2. I have added a condition check to avoid such error messages.